kolibri/core/logger/management/commands/exportlogs.py

```diff
@@ -43,81 +26,40 @@ def add_arguments(self, parser):
             action="store",
             dest="log_type",
             default="session",
-            choices=["summary", "session"],
+            choices=classes_info.keys(),
             help='Log type to be exported. Valid values are "session" and "summary".',
         )
         parser.add_argument(
             "-w",
             "--overwrite",
-            action="store",
+            action="store_true",
```

Comment:
Oh, this was an intentional change because I assumed the intention here was to just have `--overwrite` turn on the overwrite option. This is the way that I normally see binary flags done, without having to pass an argument - `store_true` means that if the flag is present, it will be set to `True`, and then without it, it defaults to `False`.

So the correct invocation now is `kolibri manage exportlogs --overwrite`.